definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-roles-and-permissions-resp-journey-nonprod.json

```diff
@@ -0,0 +1,58 @@
+[
```

Comment:
@jacmurphy can you please follow the feature-toggle file name that's already used?
I believe that's "resp-journey-roles-and-permissions".
https://github.com/hmcts/div-ccd-definitions/blob/master/definitions/divorce/json/CaseTypeTab/CaseTypeTab-resp-journey-roles-and-permissions-nonprod.json

Reply:
will do